jwst/resample/resample_spec.py

```diff
@@ -401,10 +401,10 @@ def build_interpolated_output_wcs(self, refmodel=None):
                 native2celestial = RotateNative2Celestial(ra_center_pt, dec_center_pt, 180)
                 undist2sky1 = tan | native2celestial
                 # Filter out RuntimeWarnings due to computed NaNs in the WCS
-                warnings.simplefilter("ignore")
+                with warnings.catch_warnings():
+                    warnings.simplefilter("ignore", RuntimeWarning) #was ignore. need to make more specific
```

Comment:
Does this need further updates?

Reply:
This was the third instance where the tests raised no warnings.  In this case, whoever developed this had written a comment that they wanted RuntimeWarnings filtered, so that is what I filtered.  But again, I don't know if this is the right way of going about this - should I just remove it like I did to the other ones?